src/refmterr/lib/Atom.ml

```diff
@@ -6,7 +6,7 @@
  *)
 module Range =
   struct
-    type t = ((int* int)* (int* int))
+    type t = ((int* int)* (int* int)) [@@deriving yojson]
```

Comment:
You can convert this and any other .ml files to Reason if you want

Reply:
👍